db/migrate/20201030102020_enlarge_report_status.rb

```diff
@@ -0,0 +1,25 @@
+class EnlargeReportStatus < ActiveRecord::Migration[6.0]
+  def up
+    remove_index :reports, :status
+
+    User.without_auditing do
+      Report.unscoped.find_each do |report|
+        old_calc = ConfigReportStatusCalculator.new(bit_field: report.read_attribute(:status), size: 6)
+        new_calc = ConfigReportStatusCalculator.new(counters: old_calc.status, size: 10)
+        report.update_column(:status, new_calc.calculate)
+      end
```

Comment:
We could be bit more efficient here and not initialize the AR, I don't know if it's neccessary (you tell me :)), but I got the impression `reports` can be HUGE table? :thinking: 
```suggestion
      Report.unscoped.in_batches(of: 10000) do |relation|
        relation.pluck(:id, :status).each do |report_id, current_status|
          old_calc = ConfigReportStatusCalculator.new(bit_field: current_status, size: 6)
          new_calc = ConfigReportStatusCalculator.new(counters: old_calc.status, size: 10)
          Report.unscoped.where(id: report_id).update_all(status: new_calc.calculate)
        end
      end
```

Reply:
Yeah, good that you use `in_batches` which does not load records into memory. I think batch size of 10000 is too high, I will stick with the default tho.

Reply:
oki :+1: my thinking was we load just 10000 of `[<int>, <int>]` arrays, so it should be fine, but default will work just fine as well :+1: